tracim/tracim/controllers/content.py

```diff
@@ -235,18 +234,15 @@ def get_one(self, file_id, revision_id=None):
                                  revision_id)
             preview_urls.append(url)
 
-        pdf_available = 'false'  # type: str
+        enable_pdf_buttons = False  # type: bool
         try:
             enable_pdf_buttons = \
-                preview_manager.has_pdf_preview(
-                    file_path=file_path,
-                )  # type: bool
-            pdf_available = str(enable_pdf_buttons).lower()
+                preview_manager.has_pdf_preview(file_path=file_path)
         except PreviewGeneratorException as e:
+            # INFO - A.P - Silently intercepts preview exception
+            # As preview generation isn't mandatory, just register it
             logger.debug(self, 'Exception: {}'.format(e.__str__))
-            msg_str = _('Sorry... No PDF downloads for {}: {}')  # type: str
-            msg = msg_str.format(file.file_name, str(e))  # type: str
-            tg.flash(msg, CST.STATUS_ERROR)
+        pdf_available = str(enable_pdf_buttons).lower()
```

Comment:
in this case, prefer to write explicitly what you want to have : 

```pdf_available = 'true' if enable_pdf_buttons else 'false'```

It's less dynamic but totally explicity (for example, if the API of preview_generator changes, your code may be broken)